sandbox/cgroup/cgroup.go

```diff
@@ -100,6 +103,43 @@ func Version() (int, error) {
 	return probeVersion, probeErr
 }
 
+// ProcGroup finds the path of a given cgroup controller for provided process
+// id.
+func ProcGroup(pid int, controller string) (string, error) {
+	f, err := os.Open(ProcPidPath(pid))
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		// we need to find a string like:
+		//   ...
+		//   <id>:<controller[,controller]>:/<path>
+		//   7:freezer:/snap.hello-world
+		//   ...
+		// See cgroup(7) for details about the /proc/[pid]/cgroup
+		// format.
+		l := strings.Split(scanner.Text(), ":")
+		if len(l) < 3 {
+			continue
+		}
+		controllerList := l[1]
+		cgroupPath := l[2]
+		if !strings.Contains(controllerList, controller) {
```

Comment:
Can you please add a test for the `cpu` controller and the `cpuset` controller being present.

Hint: there is a bug.